perun-core/src/main/java/cz/metacentrum/perun/core/impl/UsersManagerImpl.java

```diff
@@ -1396,11 +1397,11 @@ public void deleteSponsorLinks(PerunSession sess, User sponsor) {
 	public List<Resource> getAssignedResources(PerunSession sess, User user) {
 		try  {
 			return jdbc.query("select distinct " + resourceMappingSelectQuery + " from resources" +
-					" join groups_resources on resources.id=groups_resources.resource_id " +
+					" join groups_resources on resources.id=groups_resources.resource_id and groups_resources.status=?::group_resource_status::group_resource_status " +
```

Comment:
There seems to be an extra `::group_resource_status`.

Reply:
Fixed, thank you.